### PR TITLE
[Trivial] Shift non-Fluid specific operations to separate file

### PR DESCRIPTION
**Pull request by ghost**
This is to allow for working on the implementations for the token and document signing/timestamping systems

### What is the purpose of this pull request (PR)?
To shift non-Fluid specific operations to separate file

#### Was this PR tested and how? (If testing is not needed, please explain why)
Yes, on Ubuntu 16.04 LTS

### CODE REVIEW

File: src/script/script.h [hunk 1]
```diff
@@ -212,26 +212,6 @@ enum opcodetype
     OP_IDENTITY_UPDATE = 0xd5,
     OP_IDENTITY_MULTISIG = 0xd6,
 
-    // distributed exchange
-    OP_OFFER_ACTIVATE = 0x04,
-    OP_OFFER_UPDATE = 0x05,
-    OP_OFFER_ACCEPT = 0x06,
-    OP_OFFER_ACCEPT_FEEDBACK = 0x07,
-
-    // distributed licensing system
```
Comment (spencerlievens) [inline src/script/script.h:221]: Is it worth leaving the OP_CERT_* in the codebase and remove the rest that won't be needed in the future for identity/certs?

**Comment by spencerlievens** [comment]
utACK

File: src/script/script.h (hunk 1 above)
Comment (ghost) [reply src/script/script.h:221]: Identities is what would be immediately implemented without much change, although Certs would require a good amount of change so it wouldn't be in the immediate future, so I removed the cert OPCODES

**Comment by spencerlievens** [comment]
Could you PR towards the v2.1 branch please.

**Pull request merged**